src/Components/Footer.tsx

```diff
@@ -33,7 +33,7 @@ const Footer = () => {
           <img
             src={footer2Img}
             alt=""
-            style={{ width: "100vw", height: "auto" }}
+            style={{ width: "100%", height: "auto" }}
```

Comment:
100% did not work before? Is it working now or why did we change from 100vw?

Reply:
cause we had a scroll bar because of it, I gave the box of it 100%, and now it's working

Reply:
Great!